### PR TITLE
intial isse 4 commit

### PR DESCRIPTION
**Pull request by dipanshu231099**
this represents the work uptil now.

### CODE REVIEW

File: carpool/forms.py [hunk 1]
```diff
@@ -25,7 +24,7 @@ class Meta:
 class PoolForm(forms.ModelForm):
     user = forms.ModelChoiceField(queryset=User.objects.all(), required=False)
     tot = forms.IntegerField(widget=forms.NumberInput(), label="Seats")
-    dateTime = forms.DateTimeField(widget=forms.widgets.DateTimeInput(), label="Date Time YYYY-MM-DD HH:MM")
+    dateTime = forms.CharField(widget=forms.TextInput(attrs={'id':'datepicker-3'}))
```
Comment (Varunvaruns9) [inline carpool/forms.py:27]: This was supposed to input both the date and time of the ride.

File: carpool/templates/add.html [hunk 2]
```diff
@@ -41,6 +41,7 @@
 
     <body>
 
+    <input
```
Comment (Varunvaruns9) [inline carpool/templates/add.html]: Why is this incomplete tag here?

**Review by Varunvaruns9 — CHANGES_REQUESTED** [review]
Initial comments.

File: carpool/forms.py (hunk 1 above)
Comment (dipanshu231099) [reply carpool/forms.py:27]: yes it is

Comment (dipanshu231099) [reply carpool/forms.py:27]: the css and js for this id is different in add.html

File: carpool/templates/add.html (hunk 2 above)
Comment (dipanshu231099) [reply carpool/templates/add.html]: i removed the tag in the latest repo

File: carpool/forms.py (hunk 1 above)
Comment (Varunvaruns9) [reply carpool/forms.py:27]: Oh ok.

**Review by Varunvaruns9 — APPROVED** [review]

**Comment by Varunvaruns9** [comment]
Merged. Thanks, @dipanshu231099.

Please add a `Fixes: #<issue number>` in one of the commits from next time so that it automatically closes the related issue.

**Pull request merged**